hooked-web3-ddp-provider-client.js

```diff
@@ -5,6 +5,8 @@ export class HookedWeb3DdpProvider {
   constructor({transaction_signer}) {
```

Comment:
I'd rather not attach the `Meteor` object to the instance itself. Maybe just use the `Meteor` closure her. And for Web3 I think the recommended way nowadays is to use it like that:
```
this._web3 = new Web3();
```

and then replace `this._Web3.prototype` with just `this._web3`.